packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -1013,7 +1020,7 @@ class DebuggerController extends DisposableController
   // TODO(kenz): search through previous matches when possible.
   @override
   List<SourceToken> matchesForSearch(
-    String search, {
+    String? search, {
```

Comment:
make this non-nullable. Default callers to the empty string if they were previously passing null